runtime/protocol/http-client-engines/test-suite/build.gradle.kts

```diff
@@ -58,67 +58,80 @@ kotlin {
     }
 }
 
-open class LocalTestServers : DefaultTask() {
-    @Internal
-    var server: Closeable? = null
-        private set
-
-    @Internal
-    lateinit var main: String
+abstract class TestServerService :
```

Comment:
naming: TestServerService is wonky, can it just be called `TestServer`? `TestServerProvider`?